nuxeo-ai-core/src/test/resources/script/metadataListener.groovy

```diff
@@ -29,7 +29,7 @@ EnrichmentMetadata metadata = context.properties["enrichmentMetadata"]
 TransactionHelper.runInTransaction {
     CoreInstance.doPrivileged(metadata.context.repositoryName, { session ->
```

Comment:
why do we have a groovy listener in the first place??

Reply:
It's just a Nuxeo test listener.  The language isn't that important, although I do like Groovy.